modules/light-clients/06-solomachine/types/client_state.go

```diff
@@ -40,6 +40,18 @@ func (cs ClientState) GetLatestHeight() exported.Height {
 	return clienttypes.NewHeight(0, cs.Sequence)
 }
 
+// Status returns the status of the solo machine client.
+// The client may be:
+// - Active
+// - Frozen
```

Comment:
nit, I'd briefly define the cases for each status
```suggestion
// - Active: if frozen sequence is 0
// - Frozen: otherwise
```